src/main/java/org/mskcc/cbio/oncokb/repository/TokenStatsRepository.java

```diff
@@ -19,4 +20,10 @@
 
     @Query("select sum(tokenStats.usageCount) as count, tokenStats.token as token from TokenStats tokenStats where tokenStats.accessTime < ?1 group by tokenStats.token")
     List<UserTokenUsage> countTokenUsageByToken(Instant before);
+
+    @Query("select sum(case when tokenStats.usageCount = 0 then 1 else tokenStats.usageCount end) as count, tokenStats.token as token, DATE_FORMAT(tokenStats.accessTime,'%Y-%m') as time, tokenStats.resource as resource " + 
```

Comment:
The number of records whose usage count equals to 0, based on month. Almost every month has records' usage count equals to 0, especially the usage counts of all records are 0 before 2020-07. Will have a look at state usage count.
![image](https://user-images.githubusercontent.com/32425638/104761906-2fabdc80-5729-11eb-8a35-2d0689a30461.png)
PS, the total number of records of token stats based on month:
![image](https://user-images.githubusercontent.com/32425638/104762462-03449000-572a-11eb-8018-f661f46ad546.png)

Reply:
We investigated and realized some of the POST calls were sent with zero requests.

Reply:
@Leowisd let's don't count those are 0.